e2e/fixtures/views/SelectMetricView.ts

```diff
@@ -51,6 +51,27 @@ export class SelectMetricView extends DrilldownView {
     return this.getControls().getByPlaceholder('Filter by label values');
   }
 
+  // TODO: This can probably be better by having getByGrafanaSelector
+  // https://grafana.com/developers/plugin-tools/e2e-test-a-plugin/selecting-elements#select-field
+  async setAdHocFilter(label: string, value: string) {
```

Comment:
```suggestion
  async setAdHocFilter(label: string, operator: string, value: string) {
```

Very simple change that we'll use when testing "Include/Exclude" filtering after grouping by label.